config.py

```diff
@@ -97,7 +94,9 @@ class Config:
     EMAIL_TEMPLATE_DIR = 'email_templates/'
 
     # Flask-SQLAlchemy
+    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
     SQLALCHEMY_TRACK_MODIFICATIONS = False  # remove once this becomes the default
+    SQLALCHEMY_POOL_SIZE = 1
```

Comment:
Do we still need to limit the pool size considering that this had no effect to the database issues we had?